drivers/storage/scaleio/tests/test-run.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+# This script runs tests
+
+# set -e
+# set -x
+
+: ${CF_EC2_USER:="ec2-user"}
+
+CF_STACK_NAME="$1"
+TEST_BINARY="$2"
```

Comment:
Hi @dvonthenen,

These scripts are not going to be invoked with any command line options from Travis. They are executed as is. It's up to the script to either read environment variables or know in advance where files are. For example, the test binary is a known location. If you want to make things flexible then update the script like so:

```bash
TEST_BINARY=${TEST_BINARY:-$2}
TEST_BINARY=${TEST_BINARY:-$GOPATH/src/github.com/codedellemc/libstorage/drivers/storage/scaleio/tests/scaleio.test}
```

The above change sets the variable `TEST_BINARY` to the value of the second CLI arg (`$2`) if `$TEST_BINARY` is not already defined.

The second change sets the variable `TEST_BINARY` to the value `$GOPATH/src/github.com/codedellemc/libstorage/drivers/storage/scaleio/tests/scaleio.test`, the default path of the test binary, if the variable `$TEST_BINARY` is not already defined.